.github/workflows/release.yml

```diff
@@ -3,14 +3,82 @@ name: Release
 on:
   workflow_dispatch:
 
+
 jobs:
-  gradle:
+  # TOODO how do I ensure that nothing is running on a different commit
```

Comment:
What do you mean by this TODO? Are you worried that someone else might be concurrently running the release?